README.md

```diff
@@ -19,6 +19,7 @@ Following features and tools are supported:
 
 * 🐟 Fish Shell
 * 📷 AzCopy
+* 🔐 Bitwarden
 * ⚙️  Direnv
```

Comment:
Is there a superfluous blank before Direnv?

Reply:
Yes, you're right. Fixed.